engines/zypper_auth/spec/requests/strict_authentication/authentication_controller_spec.rb

```diff
@@ -204,6 +206,41 @@
 
         it { is_expected.to have_http_status(200) }
       end
+
+      context 'system is hybrid' do
```

Comment:
Can you add a positive test here as well?

Maybe also move all the `allows` to a before block to make clear what is test and what is test setup

Reply:
Done in [d590707](https://github.com/SUSE/rmt/pull/1202/commits/d5907077d6966225ff7feaca17ff0b7d201336a1)